OpenHPL/Functions/DarcyFriction/Friction.mo

```diff
@@ -15,9 +15,9 @@ protected
   Modelica.SIunits.ReynoldsNumber N_Re "Reynolds number";
   Real f "friction factor";
 algorithm
-  N_Re := rho * abs(v) * D / mu;
+  N_Re := rho * max(-v,v) * D / mu;
   f := fDarcy(N_Re, D, eps);
```

Comment:
Sorry, that is really not good.  This change should be reverted. If this really is a performance issue (could you give  a test case) then this is a tool issue and should not be solved in the library with work arounds.

Reply:
When i ran stand alone surge tank model i checked it in openmodelica with and without max method. How shlould i create a test case for this in OpenHPL. Or I will revert it now and maybe we fix it in future.

Reply:
In that case that would definitely be a bug in OpenModelica. So good if there is a small test case so we can run a comparison also with other tools.